xbmc/threads/Thread.h

```diff
@@ -27,6 +27,16 @@ class IRunnable;
 // minimum as mandated by XTL
 #define THREAD_MINSTACKSIZE 0x10000
 
+const int THREAD_BASE_PRIORITY_LOWRT{15};
+const int THREAD_BASE_PRIORITY_MAX{2};
+const int THREAD_BASE_PRIORITY_MIN{-2};
+const int THREAD_BASE_PRIORITY_IDLE{-15};
+const int THREAD_PRIORITY_LOWEST{THREAD_BASE_PRIORITY_MIN};
+const int THREAD_PRIORITY_BELOW_NORMAL{THREAD_PRIORITY_LOWEST + 1};
+const int THREAD_PRIORITY_NORMAL{0};
+const int THREAD_PRIORITY_HIGHEST{THREAD_BASE_PRIORITY_MAX};
+const int THREAD_PRIORITY_ABOVE_NORMAL{THREAD_PRIORITY_HIGHEST - 1};
```

Comment:
This has to be moved to `threads/platform/pthreads/ThreadImpl.h`. Windows defines that in `windows.h` which is included via `threads/platform/win/ThreadImpl.h` -> `threads/platform/ThreadImpl.h` -> `threads/Thread.h`.